GamingTheSyllabus/Assets/LoadingScreenManager.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LoadingScreenManager : MonoBehaviour
```

Comment:
Improves the flow of the game significantly. Everything is really starting to come together!